### DISCUSSION TITLE
Multi-select Picker: pass wrong arguments

### DISCUSSION THREAD

**Discussion by nhuanluong** [Budibase/budibase#8057] [Help] [answered]
hi
Someone can help,
I have a custom query like this:

> SELECT 
> 	t.name as team_name,
> 	s.name as sprint_name ,
> 	concat(t.name, ' ', s.name) as col_name ,
> 	round(tv.velocity, 2) as velocity
> FROM team_velocity tv
> LEFT JOIN sprint s ON tv.sprint_id = s.id
> LEFT JOIN team t on tv.team_id = t.id
> WHERE  tv.velocity IS NOT NULL 
> 	AND CASE WHEN FIND_IN_SET({{team_picker_str}}, 'NA') = FALSE THEN  tv.team_id IN({{team_picker}}) ELSE TRUE END
> GROUP BY tv.sprint_id, tv.team_id

In my screen, I create a form and add a provider that gets data from the custom query above, and add a Multi-select Picker
My arguments are passed to the provider as below:

team_picker : `return $("f_team.Fields.team_picker")?.join(',')`

team_picker_str: `return $('f_team.Fields.team_picker')?.join(',')?.toString()`

When I choose only one value from the picker, it works fine.
MySQL log:

> SELECT 
> 	t.name as team_name,
> 	s.name as sprint_name ,
> 	concat(t.name, ' ', s.name) as col_name ,
> 	round(tv.velocity, 2) as velocity,
> 	usr_fn_join_sprint_member_name(s.name,
> 	t.name) as short_name
> FROM team_velocity tv
> LEFT JOIN sprint s ON tv.sprint_id = s.id
> LEFT JOIN team t on tv.team_id = t.id
> WHERE  tv.velocity IS NOT NULL 
> 	AND CASE WHEN FIND_IN_SET(3, 'NA') = FALSE THEN  tv.team_id IN(3) ELSE TRUE END
> GROUP BY tv.sprint_id, tv.team_id

**But When I choose multi-value, it's not work**
the SQL log:

> SELECT 
> 	t.name as team_name,
> 	s.name as sprint_name ,
> 	concat(t.name, ' ', s.name) as col_name ,
> 	round(tv.velocity, 2) as velocity,
> 	usr_fn_join_sprint_member_name(s.name,
> 	t.name) as short_name
> FROM team_velocity tv
> LEFT JOIN sprint s ON tv.sprint_id = s.id
> LEFT JOIN team t on tv.team_id = t.id
> WHERE  tv.velocity IS NOT NULL 
> 	AND CASE WHEN FIND_IN_SET('2001-01-02 00:00:00.000', 'NA') = FALSE THEN  tv.team_id IN('2001-01-02 00:00:00.000') ELSE TRUE END
> GROUP BY tv.sprint_id, tv.team_id

I wonder why the argument is '2001-01-02 00:00:00.000', it's a date

Where did I do wrong?

I tried to pass with the handlebar `{{ f_team.Fields.team_picker }},` but the result was the same

Budibase version: 2.0.6

Thank you.

**Comment by melohagan** [comment, marked as answer]
Hi @nhuanluong 

Apologies, this is a known bug! https://github.com/Budibase/budibase/issues/7887
